pkg/controller/controller_test.go

```diff
@@ -1281,8 +1294,21 @@ func TestReconcileBindingNamespaceError(t *testing.T) {
 	testController.reconcileBinding(binding)
 
 	actions := filterActions(fakeCatalogClient.Actions())
-	if a := len(actions); a != 0 {
-		t.Fatalf("Unexpected number of actions: expected 0, got %v. Actions: %+v", a, actions)
+	if a := len(actions); a != 1 {
+		t.Fatalf("Unexpected number of actions: expected 1, got %v. Actions: %+v", a, actions)
+	}
+
+	var (
```

Comment:
so we could just get rid of it then? I'd prefer less lines of code and I think it's pretty clear from the lines below what's going on. Thoughts?